strings/charoccurrence/charoccurrence.go

```diff
@@ -0,0 +1,19 @@
+// charoccurrence.go
+// description: Chars occurrence in the sentence
+// details: An algorithm to count chars in string
+// author(s) [Moein](https://github.com/mo1ein)
+// see charoccurrence_test.go
+
+package charoccurrence
```

Comment:
This doesn't belong in a separate package - its a simple utility and specifically related to strings therefore it should be in the `strings` package.

Reply:
Do you mean put the files outside the `charoccurrence` dir and use `strings` package instead of `charoccurrence`?

Reply:
yes

Reply:
OK.